met-web/src/components/appLayouts/AuthenticatedLayout.tsx

```diff
@@ -22,7 +22,10 @@ export const AuthenticatedLayout = () => {
                 <InternalHeader />
                 <Notification />
                 <NotificationModal />
-                <Box component="main" sx={{ flexGrow: 1, marginTop: { xs: '3.5em', md: '6.5em' } }}>
+                <Box
+                    component="main"
+                    sx={{ flexGrow: 1, marginTop: { xs: '3.5em', md: '6.5em' }, width: '100%', overflowX: 'auto' }}
```

Comment:
Looks good. Was that right padding present on the header element? Maybe it was just introduced with a component.